modules/auxiliary/gather/ip_geolocate.rb

```diff
@@ -0,0 +1,103 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'json'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Rex::Proto::Http
+  include Msf::Exploit::Remote::HttpClient
+  # HttpClient is only included because of normalize_uri
```

Comment:
You probably don't even need it. Do the normalization yourself if it's really important. A full `include` for just one method is kinda icky, especially if it's all of `HttpClient` (and what it includes itself).